ras_party/controllers/account_controller.py

```diff
@@ -297,7 +301,7 @@ def request_password_change(payload, session):
                 email_address, personalisation, str(party_id))
         except RasNotifyError:
             # Note: intentionally suppresses exception
-            logger.error('Error sending notification email for party_id', party_id=party_id)
```

Comment:
Does this party_id need changed to respondent_id?

Reply:
I think this log message is a little misleading - we don't send emails from the party service. We should be explicit in our log messages, something like "Error sending request to notify gateway" would be clearer here.